daphne_worker_test/tests/test_runner.rs

```diff
@@ -634,6 +421,8 @@ impl JanusServerHandle {
     }
 }
 
+// TODO(issue #138) Re-enable Janus interop tests.
+/*
```

Comment:
I think is cleaner to use a cfg directive.
`#[cfg( feature= "janus" )]`

Reply:
I agree. But even with the directive, cargo will try to compile this function, but it doesn't currently compile. I could just remove the janus stuff altogether, then add it back when we're ready. That is, I would delete everything that is commented here and delete tests/janus.rs entirely. Would that be preferable you think?

Reply:
Ohhh that worked. Thanks! Done.